app/views/defendants/_offences.html.haml

```diff
@@ -12,6 +13,7 @@
       %th.govuk-table__header{ scope: 'col', class: 'govuk-!-width-one-quarter' }= t('search.result.offence.mode_of_trial')
   %tbody.govuk-table__body
     - decorate_each(defendant.offence_summaries, OffenceDecorator) do |offence|
+      - next if offence_ids&.exclude?(offence.id)
```

Comment:
It'd be great to exclude this from inside `defendant.offence_summaries`
Maybe using a new method inside the `defendant`?